lib/tasks/wait_for_apps.rake

```diff
@@ -0,0 +1,24 @@
+require_relative "../wait_for_healthy_container"
+
+def publishing_app_containers
+  compose = Docker::Compose.new
+  containers = compose.ps
+  containers.where do |container|
+    !WaitForHealthyContainer.container_is_service(container, "publishing-e2e-tests")
+  end
```

Comment:
if containers is an enumerable might be more idiomatic to have this where as a reject